imagetest/test_suites/storageperf/iops_write_test.go

```diff
@@ -92,12 +92,21 @@ func TestRandomWriteIOPS(t *testing.T) {
 	}
 
 	finalIOPSValue := fioOut.Jobs[0].WriteResult.IOPS
-	//TODO: Update this value to be equal to the input IOPS value, once it is implemented in this testing framework. For now, hyperdisk IOPS are the lesser of 100 IOPS per GiB of disk capacity or 350,000, if unspecified.
-	expectedHyperdiskIOPS := math.Min(100*hyperdiskSize, 350000)
-	if finalIOPSValue < iopsErrorMargin*expectedHyperdiskIOPS {
-		t.Fatalf("iops average was too low: expected close to %f, got  %f", expectedHyperdiskIOPS, finalIOPSValue)
+	expectedRandWriteIOPSString, err := utils.GetMetadataAttribute(randWriteAttribute)
+	if err != nil {
+		t.Fatalf("could not get metadata attribut %s: err %v", randWriteAttribute, err)
+	}
+
+	expectedRandWriteIOPSString = strings.TrimSpace(expectedRandWriteIOPSString)
+	var expectedRandWriteIOPS float64
+	if expectedRandWriteIOPS, err := strconv.ParseFloat(expectedRandWriteIOPSString, 64); err != nil {
+		t.Fatalf("benchmark iops string %f was not a float: err %v", expectedRandWriteIOPS, err)
 	}
-	t.Logf("iops test pass with %f iops, expected at least %f", finalIOPSValue, expectedHyperdiskIOPS)
+	if finalIOPSValue < iopsErrorMargin*expectedRandWriteIOPS {
+		t.Fatalf("iops average was too low: expected close to %f, got  %f", expectedRandWriteIOPS, finalIOPSValue)
+	}
+
+	t.Logf("iops test pass with %f iops, expected at least %f", finalIOPSValue, expectedRandWriteIOPS)
```

Comment:
This might be a confusing pass message, finalIOPSValue could be less than expectedRandWriteIOPS but still pass and print this. Printing the calculated actual minimum or printing "within margin %d of %s iops" would be more clear.

Reply:
Changed so that it prints what decimal factor (0.95 in this case) of the expected value is required to pass the test